src/libs/actions/PersonalDetails.js

```diff
@@ -123,8 +133,12 @@ function fetch() {
             const allPersonalDetails = formatPersonalDetails(data.personalDetailsList);
             Onyx.merge(ONYXKEYS.PERSONAL_DETAILS, allPersonalDetails);
 
-            const myPersonalDetails = allPersonalDetails[currentUserEmail]
-                || {avatarURL: getAvatar(undefined, currentUserEmail)};
+            myPersonalDetails = allPersonalDetails[currentUserEmail]
```

Comment:
Oh my bad I completely missed that we are reassigning this. I think maybe we don't actually need to and maybe don't even need to subscribe to `myPersonalDetails` at all here.